docs/src/components/home/AdvancedShowcase.tsx

```diff
@@ -1701,22 +1702,15 @@ export default function DataTable() {
           })}
         >
           <XGridGlobalStyles />
-          <Box
-            sx={(theme) => ({
-              textAlign: 'center',
-              py: 1,
-              position: 'relative',
-              borderBottom: '1px solid',
-              borderColor: 'grey.100',
-              ...theme.applyDarkStyles({
-                borderColor: 'primaryDark.600',
-              }),
-            })}
+          <Typography
+            variant="body2"
+            color="text.primary"
+            fontWeight="semiBold"
+            sx={{ position: 'relative', textAlign: 'center', py: 1.5 }}
           >
-            <Typography color="primary.main" fontWeight={700}>
-              Trades, October 2020
-            </Typography>
-          </Box>
+            Trades, October 2020
```

Comment:
Searching for where is used this element let me notice that the hover color of the chip is a bit weird, because there is no on-click event. you can do a double click on the cell but nothing special on the chip.

Clearly out of the scope of this PR.

![image](https://github.com/mui/material-ui/assets/45398769/6424f599-5350-4c29-8e4c-ec9647157255)

Reply:
Yeah — but it's a good thing you called out. There are three instances of the Data Grid on this page, and they're all slightly differently customized. For the next one, I might tackle this to make sure they're all consistent, regardless of whether they're using different demo datasets!